### DISCUSSION TITLE
Forms: why so many classes predefined and not changable?

### DISCUSSION THREAD

**Discussion by ekandreas** [filamentphp/filament#2448] [General]
In the latest release the repeated got like 100 new classes predefined (the button, rows and border).
We currently work hard with not putting blue buttons inside dialoges and we don't want rounded borders and buttons.
But now there are several classes already defined not possible to change.
Why is that? And how can we make filament forms more adaptable to different solutions?

**Comment by danharrin** [comment]
I'm open to suggestions on improving the customisation process, but unfortunately that is not a priority when we are introducing new features.

Ideally I'd like classes such as `filament-forms-rounding` that can be used to customize the rounding of all inputs at once, for example.
